parameter/FloatingPointParameterType.cpp

```diff
@@ -138,7 +139,8 @@ bool CFloatingPointParameterType::toBlackboard(
             return false;
         }
 
-        float fData = reinterpret_cast<const float &>(uiValue);
+        float fData;
+        utility::binaryCopy(uiValue, fData);
```

Comment:
Lets pretend that binary returns the copy:
~~~cpp
float fData = utility::binaryCopy<float>(uiValue)
~~~
I prefer to initialize variable on declaration.